helm/thingsboard/values.yaml

```diff
@@ -0,0 +1,285 @@
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+http:
+  # kind can be either Deployment or StatefulSet
+  kind: StatefulSet
+  image:
+    repository: thingsboard/tb-http-transport
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8082
+    protocol: TCP
+    name: http
+  timeout: 10000
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8082
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+coap:
+  # kind can be either Deployment or StatefulSet
+  kind: StatefulSet
+  image:
+    repository: thingsboard/tb-coap-transport
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8083
+    protocol: UDP
+    name: coap
+  timeout: 10000
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8083
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+webui:
+  # kind can be either Deployment or StatefulSet
+  kind: Deployment
+  image:
+    repository: thingsboard/tb-web-ui
+    # Overrides the global image values
+    #server: docker.io
+    #pullPolicy: Always
+    #tag: "X.Y.Z"
+  replicaCount: 1
+  port:
+    number: 8084
+    protocol: TCP
+    name: http
+  podAnnotations: {}
+  podSecurityContext: {}
+  securityContext: {}
+  service:
+    type: ClusterIP
+    port: 8084
+  resources: {}
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    # targetMemoryUtilizationPercentage: 80
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+cassandraInitDB:
+  keyspace: thingsboard
+  replication:
+    class: SimpleStrategy
+    factor: 1
+
+cassandra:
+  # Database can be either postgres or hybrid (entities in postgres and timeseries in cassandra)
+  # for an postgresql database set cassandra.enabled as false
+  # for an hybrid database set cassandra.enabled as true
+  enabled: false
+  replicaCount: 1
+  dbUser:
+    password: setplease
+
+postgresql-ha:
+  nameOverride: postgresql-ha
+  postgresql:
+    database: thingsboard
+    replicaCount: 1
+    password: setplease
+    repmgrPassword: setplease
+  postgresqlImage:
+    tag: 12
+  pgpool:
+    adminPassword: setplease
+    replicaCount: 1
+  pgpoolImage:
+    tag: 4
```

Comment:
what is this thing?

Reply:
By default bitnami chart uses pg version 11 and thingsboard uses a single instance of postgres, not a chart dependency, and using version 12.
I wanted to keep same version as upstream is using (12) but still use the bitnami helm chart as dependency...
If I bump version of pg to 12 I think it is safer also bumping pgpool image version to the latest one. But no more tests done than creating a device and sending telemetries. it works, but I'm not sure it it is safer or just leaving the default previous version if using pg 12.
Any suggestion on this?

Reply:
If it works, than it should be fine.